arangodb-net-standard.Test/Serialization/JsonNetApiClientSerializationTest.cs

```diff
@@ -182,7 +210,10 @@ public void Serialize_ShouldNotCamelCaseParams_WhenSerializingPostTransactionBod
 
             var serialization = new JsonNetApiClientSerialization();
 
-            byte[] jsonBytes = serialization.Serialize(body, new ApiClientSerializationOptions(true, true));
+            byte[] jsonBytes = serialization.Serialize(body, new ApiClientSerializationOptions(
+                 useCamelCasePropertyNames: true,
+                 ignoreNullValues: true,
+                 camelCasePropertyNamesOfObjectValuesInDictionaries: false));
```

Comment:
Removing `camelCasePropertyNamesOfObjectValuesInDictionaries` here would show that backward compatibility is preserved thanks to the default value being false.

Reply:
I've removed the unnecessary code.